dokan/dokan.h

```diff
@@ -136,23 +137,23 @@ typedef struct _DOKAN_FILE_INFO {
   /** A pointer to DOKAN_OPTIONS which was passed to DokanMain. */
   PDOKAN_OPTIONS DokanOptions;
   /**
-   * Process id for the thread that originally requested a given I/O operation
+   * Process id for the thread that originally requested a given I/O operation.
    */
   ULONG ProcessId;
   /**
-   * Requesting a directory file
-   * IsDirectory has to be set in CreateFile is the file appear to be a folder.
+   * Requesting a directory file.
+   * Must be set in \ref DOKAN_OPERATIONS#ZwCreateFile if the file appear to be a folder.
```

Comment:
There is a difference with DOKAN_OPERATIONS#ZwCreateFile and DOKAN_OPERATIONS.ZwCreateFile ?
Both seems to work :D